### PR TITLE
Only add signed urls for `secure.notion-static.com` urls

### LINKED ISSUE #331: NotionAPI getSignedfileUrls error HTTPError: Response code 400 (Bad Request)
#### Description
All the static builds were fine up until 22nd July 2022, but since yesterday, I have started getting these errors in my build, and oddly, my development builds have become super slow.

This is the log I get while building both static production and SSR development builds.
```
NotionAPI getSignedfileUrls error HTTPError: Response code 400 (Bad Request)
    at Request.<anonymous> (/home/runner/work/pipe-runner-website/pipe-runner-website/node_modules/got/dist/source/as-promise/index.js:117:42)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) ***
  code: undefined,
  timings: ***
    start: 1658648256775,
    socket: 1658648256776,
    lookup: 1658648256785,
    connect: 1658648256794,
    secureConnect: 1658648256804,
    upload: 1658648256804,
    response: 1658648256918,
    end: 1658648256918,
    error: undefined,
    abort: undefined,
    phases: ***
      wait: 1,
      dns: 9,
      tcp: 9,
      tls: 10,
      request: 0,
      firstByte: 114,
      download: 0,
      total: 1[43](https://github.com/Pipe-Runner-Lab/pipe-runner-website/runs/7486332222?check_suite_focus=true#step:7:44)
    ***
  ***
***
```

#### Notion Test Page ID

As far as I can tell, this is not a page-specific issue since earlier, the same pages have been working fine for me.
Also, I have done a shallow check and it seems the pages are rendered all right, but I am not sure yet.

#### Version

"notion-client": "5.0.0",
"notion-types": "5.0.0",
"notion-utils": "5.0.0",
"react-notion-x": "5.0.0"

Has something changed from Notion again?? 😢

### PR DESCRIPTION
**Pull request by remorses**
#### Description

Only add signed urls for `secure.notion-static.com` urls

Fix #331 

#### Notion Test Page ID

https://brave-iberis-6ea.notion.site/Videos-5af3d63c840e458e92c09c2ec6cd71b3
https://react-notion-x-demo.transitivebullsh.it/5af3d63c840e458e92c09c2ec6cd71b3

### CODE REVIEW

File: packages/notion-client/src/notion-api.ts [hunk 1]
```diff
@@ -235,7 +235,7 @@ export class NotionAPI {
         // console.log(block, source)
 
         if (source) {
-          if (source.indexOf('youtube') >= 0 || source.indexOf('vimeo') >= 0) {
+          if (!source.includes('secure.notion-static.com')) {
```
Comment (remorses) [inline packages/notion-client/src/notion-api.ts:238]: I always only encountered urls with `secure.notion-static.com` that needed to be signed, there may be more endpoints than this one

**Comment by pbteja1998** [comment]
If you remove line 229, everything will work.

Remove `block.type === 'page'`

**Comment by normdoow** [comment]
This fixed some issues for me as well where uploaded videos weren't showing up.

Like this page for example: 849918ea9a9144ba9b84d70d512274ab

Trying to understand the full scope of changes that this will have before merging in.

Thanks!

**Comment by normdoow** [comment]
I've done some more testing and this seems to work well and not break anything.
067dd719a912471ea9a3ac10710e7fdf

**Comment by transitive-bullshit** [comment]
I think this is the best fix for the issue. Thanks @remorses 🙏

**Pull request merged**